djgeojson/serializers.py

```diff
@@ -14,8 +14,15 @@
 
 from six import string_types, iteritems
 
+import django
+
 from django.db.models.base import Model
-from django.db.models.query import QuerySet, ValuesQuerySet
+
+try:
+    from django.db.models.query import QuerySet, ValuesQuerySet
+except ImportError:
+    from django.db.models.query import QuerySet
```

Comment:
Yeah that's what I had in mind.

If you don't like it (*it's a friday night idea after all...*) then you can also do something like that:

```python
try:
    from django.db.models.query import QuerySet, ValuesQuerySet
except ImportError:
    from django.db.models.query import QuerySet
    ValuesQuerySet = None
```

And then test with `if ValuesQuerySet is None` instead of `django.VERSION >= (1, 9)`.